sdk/monitor/test-resources.bicep

```diff
@@ -69,6 +71,104 @@ resource metricReaderRole 'Microsoft.Authorization/roleAssignments@2018-01-01-pr
   }
 }
 
+resource table 'Microsoft.OperationalInsights/workspaces/tables@2022-10-01' = {
+  name: tableName
+  parent: primaryWorkspace
+  properties:{
+    totalRetentionInDays: 30
+    plan: 'Analytics'
+    schema: {
+      name: tableName
+      displayName: tableName
+      description: 'Table for Ingestion testing'
```

Comment:
Also, now that we are separating the pipelines how would the shared test resources work? Do we need to do anything extra to make sure this continues to happen?

Reply:
The test-resources.bicep file is under the monitor/  directory and is called with "./eng/common/TestResources/New-TestResources.ps1 monitor" command. I think separating the pipelines should not have an affect on this but maybe we need  three test-resources.bicep files?

Reply:
I think your understanding is correct (a common test resources file will likely work). But it would be best to confirm with the engsys team.

Reply:
I think things should still work as we [pass the service directory](https://github.com/Azure/azure-sdk-for-net/blob/main/eng/pipelines/templates/jobs/live.tests.yml#L110) when deploying the resources, but I think the difference is that we will now be creating the same set of resources 3 times. Only 2 of the test pipelines require the resources, and of those 2 each one only uses a subset.

Reply:
We might need to update the New-TestResources.ps1 script to allow passing an optional library directory if we want to avoid the test resource duplication.

Reply:
I think ensuring that each pipeline runs on a staggered schedule is likely the easiest solution here and good enough to prevent conflict.

Reply:
Conflict isn't the issue - each run will use a random ID as part of the resource group name. The issue is unnecessary resource creation.

Reply:
That said, I would agree it is not something that is a blocking issue. We could probably address it in MQ.

Reply:
Created https://github.com/Azure/azure-sdk-tools/issues/4450